pkg/gardener/shoot/hyperscaler/azure/config.go

```diff
@@ -48,6 +48,12 @@ func NewInfrastructureConfig(workerCIDR string, zones []string) InfrastructureCo
 	// All Azure shoots are zoned.
 	// No zones - the shoot configuration is invalid.
 	// We should validate the config before calling this function.
+	azureZones, err := generateAzureZones(workerCIDR, zones)
+	if err != nil {
+		// TODO - pass the error to caller?
```

Comment:
Please do not forget that the error should be handled

Reply:
Fixed